app/grandchallenge/algorithms/forms.py

```diff
@@ -100,9 +100,6 @@ class JobCreateForm(SaveFormInitMixin, Form):
     creator = ModelChoiceField(
         queryset=None, disabled=True, required=False, widget=HiddenInput
     )
-    time_limit = IntegerField(
```

Comment:
Removed as this didn't need to be on the form, we still have some hardcoded things in the `form_valid` method. I think this should be a create view that manipulated the object in the forms `save` method if necessary.